Metadata.Rmd

```diff
@@ -0,0 +1,660 @@
+But if your stated requirements have a less solid rationale, this may be unnecessarily conservative and inconvenient.
+
+In the absence of clear, hard requirements, you should set minimum versions (or not) based on your expected user base, the package versions they are likely to have, and a cost-benefit analysis of being too lax versus too conservative.
+
+### Other dependencies
+
+There are three other fields that allow you to express more specialised dependencies:
+
+* `Depends`: Prior to the roll-out of namespaces in R 2.14.0 in 2011, `Depends`
+  was the only way to "depend" on another package.
+  Now, despite the name, you should almost always use `Imports`, not `Depends`.
+  You'll learn why, and when you should still use `Depends`, in
+  [namespaces](#namespace).
+    
+  You can also use `Depends` to state a minimum version for R itself, e.g.
+  `Depends: R (>= 4.0.0)`.
+  Again, think carefully if you do this.
+  This raises the same issues as setting a minimum version for a package you
+  depend on, except the stakes are much higher when it comes to R itself.
+  Users can't simply consent to the necessary upgrade, so, if other packages
+  depend on yours, your minimum version requirement for R can cause a cascade of
+  package installation failures.
+
+  - The [backports package](https://cran.r-project.org/package=backports) is
+    useful if you want to use a function like `tools::R_user_dir(()`, which was
+    introduced in 4.0.0 in 2020, while still supporting older R versions.
+  - The tidyverse packages officially support the current R version, the devel
+    version, and four previous versions.
+    We proactively test this support in the standard build matrix we use for
+    continuous integration.
+  - Packages with a lower level of use may not need this level of rigour.
+    The main takeaway is: if you state a minimum of R, you should have a reason
+    and you should take reasonable measures to test your claim regularly.      
+    
+* `LinkingTo`: packages listed here rely on C or C++ code in another package. 
+  You'll learn more about `LinkingTo` in [compiled code](#src).
+    
+* `Enhances`: packages listed here are "enhanced" by your package.
+  Typically, this means you provide methods for classes defined in another
+  package (a sort of reverse `Suggests`).
+  But it's hard to define what that means, so we don't recommend using
+  `Enhances`.
+    
+You can also list things that your package needs outside of R in the `SystemRequirements` field.
+But this is just a plain text field and is not automatically checked.
+Think of it as a quick reference; you'll also need to include detailed system requirements (and how to install them) in your README.
+
+<!-- This description of SystemRequirements seems a bit too dismissive or wishy-washy now, given the importance of this field to RSPM, ubuntu-based CI, etc. Should we rephrase? -->
```

Comment:
I think discussion can probably mostly occur in the compiled code character, since that's mostly what needs it.